docs/docsite/rst/user_guide/playbooks_python_version.rst

```diff
@@ -1,29 +1,26 @@
 .. _pb-py-compat:
 
-Python Version and Templating
-=============================
+********************
+Python3 in templates
+********************
 
-Jinja2 templates leverage Python data types and standard functions.  This
-makes for a rich set of operations that can be performed on data.  However,
-this also means that certain specifics of the underlying Python becomes
-visible to template authors.  Since Ansible playbooks use Jinja2 for templates
-and variables, this means that playbook authors need to be aware of these
-specifics as well.
+Ansible uses Jinja2 to leverage Python data types and standard functions in templates and variables.
+This approach makes for a rich set of operations that can be performed on data. However,
```

Comment:
Might be an idea to revise to active voice. e.g. "You can use this approach to perform a rich set of operations on data."